.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "grammarly.selectors": [
+    {
+      "language": "markdown",
+      "scheme": "file"
+    }
+  ]
+}
```

Comment:
Linters complaining about no new line at eof here 🤮

Reply:
Why is this file even included?

Reply:
Its gone

Reply:
yoiu're starting to sound like me 🤣